### DISCUSSION TITLE
`max-lines suppression should not need to be the first line in the file` Issue comes up in google search and implies it was never fixed

### DISCUSSION THREAD

**Discussion by Stan-Stani** [eslint/eslint#18791] [General]
https://github.com/eslint/eslint/issues/12111

Solution: Modify this issue to link to the pr that actually resolved this issue.

**Comment by nzakas** [comment]
If you can find the commit that fixed this, we can add it to the issue. However, it's likely that it was fixed incidentally in some other commit and that's why it's not linked to the issue.
